global-script.lua

```diff
@@ -7,6 +7,7 @@ minorPowerZone = "cb16ab"
 minorPowerDiscardZone = "55b275"
 majorPowerZone = "089896"
 majorPowerDiscardZone = "eaf864"
+uniquePowerDiscardZone = "uniquePowerDiscard"
```

Comment:
why does this just have a string for it's value?

Reply:
"guid" can actually be fairly arbitrary strings, but TTS generates random 6 hex-digit ones by default. I used `uniquePowerDiscard` as the GUID for the new zone. This would also be useful to do for some of the other objects that have scripts (the `aidBoard` for example).